tls/s2n_early_data.c

```diff
@@ -82,3 +85,19 @@ int s2n_psk_set_context(struct s2n_psk *psk, const uint8_t *context, uint16_t si
     memcpy_check(context_blob->data, context, size);
     return S2N_SUCCESS;
 }
+
+S2N_RESULT s2n_early_data_config_clone(struct s2n_psk *new_psk, struct s2n_early_data_config *old_config)
+{
```

Comment:
If we don't plan to make this API public for our customers, we could make it a static function.

Reply:
This method can't be static because it's used in s2n_psk.c. Are you suggesting moving the method to s2n_psk.c? Because I could, but that kind of violates the separation of concerns here.

Reply:
Yes, I was suggesting moving it to `s2n_psk.c` as I don't see it being used elsewhere and us making it a public API.

Reply:
Yes, I was suggesting moving it to `s2n_psk.c` as I don't see it being used elsewhere and we don't plan on making it a public API.

Reply:
It deals with early data configuration, so I'd rather it lived with all the other early data code. We don't really gain anything from making it static in a different file.